dbt/adapters/athena/lakeformation.py

```diff
@@ -24,6 +26,7 @@ class LfTagsConfig(BaseModel):
     enabled: bool = False
     tags: Optional[Dict[str, str]] = None
     tags_columns: Optional[Dict[str, Dict[str, List[str]]]] = None
+    inherited_tags: Optional[List[str]] = []
```

Comment:
```suggestion
    inherited_tags: List[str] = []
```
it's not Optional once it's default value is not None

Reply:
Thanks!